rdr_service/dao/participant_summary_dao.py

```diff
@@ -739,24 +739,23 @@ def update_enrollment_status(self, summary: ParticipantSummary, session, allow_d
             participant_id=summary.participantId
         )
 
-        enrollment_info = EnrollmentCalculation.get_enrollment_info(
-            EnrollmentDependencies(
-                consent_cohort=summary.consentCohort,
-                primary_consent_authored_time=summary.consentForStudyEnrollmentFirstYesAuthored,
-                gror_authored_time=summary.consentForGenomicsRORAuthored,
-                basics_authored_time=summary.questionnaireOnTheBasicsAuthored,
-                overall_health_authored_time=summary.questionnaireOnOverallHealthAuthored,
-                lifestyle_authored_time=summary.questionnaireOnLifestyleAuthored,
-                earliest_ehr_file_received_time=summary.ehrReceiptTime,
-                earliest_mediated_ehr_receipt_time=summary.firstParticipantMediatedEhrReceiptTime,
-                earliest_physical_measurements_time=earliest_physical_measurements_time,
-                earliest_biobank_received_dna_time=earliest_biobank_received_dna_time,
-                ehr_consent_date_range_list=ehr_consent_ranges,
-                dna_update_time=revised_consent_time,
-                earliest_core_physical_measurement_time=min_or_none(meas.finalized for meas in core_measurements),
-                wgs_sequencing_time=wgs_sequencing_time
-            )
+        enrollment_dependencies = EnrollmentDependencies(
+            consent_cohort=summary.consentCohort,
+            primary_consent_authored_time=summary.consentForStudyEnrollmentFirstYesAuthored,
+            gror_authored_time=summary.consentForGenomicsRORAuthored,
+            basics_authored_time=summary.questionnaireOnTheBasicsAuthored,
+            overall_health_authored_time=summary.questionnaireOnOverallHealthAuthored,
+            lifestyle_authored_time=summary.questionnaireOnLifestyleAuthored,
+            earliest_ehr_file_received_time=summary.ehrReceiptTime,
+            earliest_mediated_ehr_receipt_time=summary.firstParticipantMediatedEhrReceiptTime,
+            earliest_physical_measurements_time=earliest_physical_measurements_time,
+            earliest_biobank_received_dna_time=earliest_biobank_received_dna_time,
+            ehr_consent_date_range_list=ehr_consent_ranges,
+            dna_update_time=revised_consent_time,
+            earliest_core_physical_measurement_time=min_or_none(meas.finalized for meas in core_measurements),
```

Comment:
I think this can potentially be addressed in a separate PR if it's an actual issue.  I assume core_measurements is the required height/weight values, but I'm wondering how this works if there are cases where the height and weight values were submitted in different `physical_measurements` payloads with different finalized time (e.g., a remote PM rec with just a height followed later by an in-clinic PM rec with a weight as well).  Does it need to be the max of those two finalized dates?

Reply:
I guess I was imagining that the height and weight would need to be measured at the same time (in the same measurement), but that's a good point. I'll ping Mara in the ticket.